pkg/neorpc/result/application_log.go

```diff
@@ -57,6 +57,9 @@ func (l *ApplicationLog) UnmarshalJSON(data []byte) error {
 	} else {
 		return errors.New("no block or transaction hash")
 	}
+	if len(aux.Executions) > 2 { // OnPersist+PostPersist for block at the worst case.
+		return fmt.Errorf("invalid Executions count: got %d, allowed 2 at max", len(aux.Executions))
+	}
```

Comment:
@roman-khimov, we actually have a lot of cases like that in our unmarshallers, e.g. block, transaction, etc. Do we need to restrict all of them? C# node restricts only maximum allowed JSON depth (see [2912](https://github.com/neo-project/neo/pull/2912/files] and [827](https://github.com/neo-project/neo-modules/pull/827/files)) and for us it's not that easy to dom because neither standard JSON nor ordered JSON support nested JSON depth. But this limit is restricted by default by `10000`, see https://github.com/golang/go/issues/31789 and https://github.com/golang/go/commit/84afaa9e9491d76ea43d7125b336030a0a2a902d.

Reply:
This particular check seems to be excessive. If I want to flood you, I'll just add more of unlimited items (like notifications).

Reply:
Removed.